stable/kong/templates/controller-rbac-cluster-role-binding.yaml

```diff
@@ -17,3 +17,24 @@ subjects:
     name: {{ template "kong.serviceAccountName" . }}
     namespace: {{ .Release.Namespace }}
 {{- end -}}
+{{- if .Values.podSecurityPolicy.enabled }}
```

Comment:
Since this binding doesn't have anything specific to do with the Controller, please move to the same file as PSP and guard the entire file with the `if`.

Reply:
ok